trunner/test_runner.py

```diff
@@ -35,9 +36,10 @@ def search_for_tests(self):
     def parse_tests(self):
         self.test_configs = []
         for path in self.test_paths:
-            args = ParserArgs(yaml_path=path, targets=self.targets)
+            args = ParserArgs(yaml_path=path, targets=self.targets, long_test=self.long_test)
             config = TestCaseConfig.from_yaml(args)
-            self.test_configs.extend(config.tests)
+            for test_config in config.tests:
+                self.test_configs.append(test_config)
```

Comment:
there is no need to change it

Reply:
Thanks, changes applied in the last force push.